acquia_lift.admin.inc

```diff
@@ -287,13 +287,36 @@ function acquia_lift_admin_form_submit($form, &$form_state) {
  * @param stdClass $option_set
  *   (optional) The content variation to show in the default view.
  */
-function acquia_lift_report($form, &$form_state, $agent_data, $option_set = NULL) {
+function acquia_lift_report($form, &$form_state, $agent_data, $option_set = NULL, $nested = FALSE) {
   if (!$agent = personalize_agent_load_agent($agent_data->machine_name)) {
```

Comment:
This seems weird and I can't remember why it's there.  Does it make sense to you?